cmake/FindJSONCPP.cmake

```diff
@@ -1,44 +0,0 @@
-# - Try to find JSONCPP
-# Once done this will define
-#
-#  JSONCPP_FOUND - system has JSONCPP
-#  JSONCPP_INCLUDE_DIRS - the JSONCPP include directory
-#  JSONCPP_LIBRARY - Link these to use JSONCPP
-
-FIND_LIBRARY (JSONCPP_LIBRARIES NAMES jsoncpp
-    PATHS
-    /usr/lib
-    /usr/local/lib
-    ${CMAKE_SOURCE_DIR}/win32-deps/lib
-)
-
-FIND_PATH (JSONCPP_INCLUDE_DIRS json.h
-    PATHS
-    /usr/include
-    /usr/local/include
-    ${CMAKE_SOURCE_DIR}/win32-deps/include
-    PATH_SUFFIXES jsoncpp/json json
-)
-
-IF(JSONCPP_INCLUDE_DIRS AND JSONCPP_LIBRARIES)
-    SET(JSONCPP_FOUND TRUE)
-ENDIF(JSONCPP_INCLUDE_DIRS AND JSONCPP_LIBRARIES)
-
-IF(JSONCPP_FOUND)
-    if(EXISTS "${JSONCPP_INCLUDE_DIRS}/version.h")

```

Comment:
These checks are required for providing old jsoncpp support (version < 0.7.x). Please revert those changes.